.github/workflows/npm_publish_light.yml

```diff
@@ -0,0 +1,126 @@
+name: Build and Publish Cohort SDK NPMs
+on:
+    #workflow_call:
+    push:
+env:
+    SDK_JS_PACKAGE_NAME: "@kindredgroup/cohort_sdk_js"
+    SDK_CLIENT_PACKAGE_NAME: "@kindredgroup/cohort_sdk_client"
+jobs:
+    npm:
+        strategy:
+            fail-fast: false
+            matrix:
+                settings:
+                  - host: macos-latest
+                    dirSdkJs: packages/cohort_sdk_js
+                    dirSdkJsClient: cohort_sdk_client
+                    dirCohortInitiator: cohort_banking_initiator_js
+                    dirCohortReplicator: cohort_banking_replicator_js
+                    customiseHostCommand: echo "No customizations needed"
+
+        name: Publish SDK NPS with bundled Rust for node@16
+        runs-on: ${{ matrix.settings.host }}
+        steps:
+            - uses: actions/checkout@v3
+
+            - name: Setup node
+              uses: actions/setup-node@v3
+              with:
+                  node-version: 16
```

Comment:
Probably use `16` and `18`

Reply:
We have ticket to do testing with node 18. If we are going to support both then the whole workflow should be different in the way that it should be publishing two artefacts, one for v16 and one for v18. I prefer to have that work done as separate ticket (if we need to do it)